app/src/main/res/layout/activity_add_new_device.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.wear.widget.BoxInsetLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
this layout use for wearable UI creation

Reply:
removed boxInsetLayout